index.html

```diff
@@ -0,0 +1,30 @@
+<html><head>
+  <title>Grocery List</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div id="main">
+  <img src="grocery-bags.png">
+  <div id="app">
+    <h1>Grocery List</h1>
+    <div>
+      <input value="">
+      <input value="">
```

Comment:
we shoulde have explicit types on input tags. i.e. `<input type="text" value="" >`